match/lib/match/runner.rb

```diff
@@ -254,6 +254,24 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
         end
       end
 
+      if params[:include_all_certificates]
+        certificate_id = nil
```

Comment:
Can we add a comment above this explaining why its being set to nil? I thought this was initializing something and was confused why it wasn't used down further 😅

```suggestion
        # Clearing specified certificate id which will prevent a profile being created with only one certificate
        certificate_id = nil
```